.vib/ruby/goss/ruby.yaml

```diff
@@ -0,0 +1,16 @@
+command:
+  check-installed-gems:
+    exec: gem list
+    exit-status: 0
+    stdout:
+      {{ range $module := .Vars.modules }}
+      - {{ $module }}
+      {{ end }}
+  check-ssl:
+    exec: ruby -rnet/https -e "Net::HTTP.get URI('https://github.com')"
+    exit-status: 0
+  check-eaccess:
+    exec: grep HAVE_EACCESS -r /opt/bitnami/ruby/include/
```

Comment:
I would include a check for `HAVE_LIBPTHREAD` as well, as it is one of the compilation flags and may affect performance